### ISSUE TITLE
Integration has likely been deprecated?

### ISSUE THREAD

**Issue by mist0706** [klali/ha-plejd#63]
Looks like homeassistant expected some additional variable that is not being provided? Has this integration completely stopped working?
```
Logger: homeassistant.components.light
Source: custom_components/plejd/light.py:519
Integration: Light (documentation, issues)
First occurred: 4:39:14 PM (1 occurrences)
Last logged: 4:39:14 PM

Error while setting up plejd platform for light
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 353, in _async_setup_platform
    await asyncio.shield(task)
  File "/config/custom_components/plejd/light.py", line 519, in async_setup_platform
    hass.services.async_register("plejd", "write_data", plejd_service)
                                                        ^^^^^^^^^^^^^
NameError: name 'plejd_service' is not defined

```

**Comment by jmderander** [comment]
It could possibly be broken in later versions. I am running on homeassistant version 2023.3.1 which is working fine. You could try with that or perhaps have a look at https://github.com/thomasloven/hass_plejd/

**Comment by klali** [comment]
was a bug in commits adding a service, fixed by a2fe08ce8b9c7248dfeeab14c533152c02b9a0da
